integration-tests/integration/globalsearch/globalSearch.cy.js

```diff
@@ -35,41 +35,45 @@ context('Global search', () => {
     })
   })
 
-  it('should call global search with offender number search', () => {
-    cy.task('stubGlobalSearch')
-    const globalSearchPage = GlobalSearchPage.verifyOnPage()
-    const form = globalSearchPage.form()
-    form.search().clear().type('A1234BC')
-    form.submitButton().click()
-
-    GlobalSearchPage.verifyOnResultsPage()
-
-    cy.task('verifyGlobalSearch').should((requests) => {
-      expect(requests).to.have.lengthOf(1)
-      expect(JSON.parse(requests[0].body)).to.deep.equal({
-        prisonerIdentifier: 'A1234BC',
-        gender: 'ALL',
-        location: 'ALL',
-        includeAliases: true,
-      })
+  context('should call global search', () => {
+    beforeEach(() => {
+      cy.visit('/global-search')
+      cy.task('stubGlobalSearch')
```

Comment:
do we need to stub before visiting? seems like this would be a bit of a race condition?